downstream/attributes/attributes.adoc

```diff
@@ -17,6 +17,7 @@
 :VMBase: VM-based installation
 :OperatorBase: operator-based installation
 :ContainerBase: container-based installation
+:PlatformUI: platform dashboard
```

Comment:
Hey @ariordan-redhat ! I think you might want to call this PlatformDashboard because there is discussion around terminology for using platform UI instead of platform gateway in the future and as named, this might cause confusion. Otherwise, I think this looks good.